pkg/detectors/elasticcloud/elasticcloud.go

```diff
@@ -0,0 +1,101 @@
+package elasticcloud
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+
+	keyPat = regexp.MustCompile(`\b(essu_(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{4}|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{2}={2}))`)
+)
+
+func (s Scanner) Keywords() []string {
+	return []string{"elasticcloud", "elastic-cloud", "apikey", "essu"}
```

Comment:
I think `apikey` is a very common term and we should not use it as Keyword for a specific detector. We should use Keywords related to detector.

Reply:
removed in ee84e19